app/models/appointment_summary.rb

```diff
@@ -53,9 +54,12 @@ def postcode=(str)
               message: '%{value} is not a valid organisation'
             }
 
-  validates :address_line_1, presence: true
-  validates :town, presence: true
-  validates :postcode, presence: true, postcode: true
+  validates :address_line_1, presence: true, length: { maximum: 50 }
+  validates :address_line_2, length: { maximum: 50 }
+  validates :address_line_3, length: { maximum: 50 }
+  validates :town, presence: true, length: { maximum: 50 }
+  validates :county, length: { maximum: 50 }
+  validates :postcode, presence: true, postcode: true, length: { maximum: 50 }
```

Comment:
We don't need to validate length of postcode. The postcode validation will handle this.